Create MulticulturalMesh.pdf

TimTor opened the pull request:
Here is a pdf to review, edit approve for hand out tomorrow at Multicultural Mesh and perhaps in future events.

TimTor committed:
Create MulticulturalMesh.pdf

dasanchez commented:
@TimTor solid!
Can you rename the folder to follow the lowercase and dashes scheme and upload the source file please?

TimTor committed:
Handout for events - Basic ID

This handout is four to a page with basic online contact links for ToMesh and Free Geek.  Also a brief mission statement, current at 2019, with elaboration.

TimTor committed:
file naming norms

lower case and dashes

dasanchez commented:
I think it's ready for printing!
Can you upload the source file too?

TimTor committed:
Create MulticulturalMesh.docx

Added the design source file (MS Word used)

Shrinks99 reviewed (approved):
Will approve & merge for now however it seems the PDF uses rasterized fonts and stuff instead of vectors.  Probably Microsoft Word's font.  These should be redone at some point at a higher quality however until we have better branding that's not a problem IMO.

Thanks for these @TimTor!
